Neo4jClient/Neo4jClient.nuspec

```diff
@@ -11,7 +11,7 @@
     <tags>neo4j nosql cypher</tags>
     <dependencies>
       <dependency id="Microsoft.Net.Http" />
-      <dependency id="Newtonsoft.Json" version="6.0.3" />
+      <dependency id="Newtonsoft.Json"  />

```

Comment:
If changing to 6.0.8 is a required change (as above), then this should read `version="6.0.8"` to force the use of Json.NET 6.0.8 or higher.